gui/apitracemodel.cpp

```diff
@@ -75,7 +75,7 @@ QVariant ApiTraceModel::data(const QModelIndex &index, int role) const
                         ", %1kb";
 
                 ApiFramebuffer fbo = frame->state()->colorBuffer();
-                QImage thumb = fbo.thumb();
+                QImage thumb = fbo.calculateThumbnail(false, false);
```

Comment:
There also is a frame->thumbnail which comes from gathering the thumbnails from retrace. 

This smells like duplicated functionality but I am not sure.